### ISSUE TITLE
Improve LWRP docs.

### ISSUE THREAD

**Issue by jonlives** [chef-boneyard/chef-web-docs-2016#499]
Per https://github.com/chef/chef-rfc/pull/102, the suggestion has been made that we look at improving the LWRP documentation, particularly around updated_by_last_action and use_inline_resources. This issue is to capture that request, and discuss what form the improvements might take.

**Comment by someara** [comment]
I can help with this for sure.. commenting here to remind myself =)

**Comment by jamescott** [comment]
Step one: consolidate the topics about the `use_inline_resources` method:

https://github.com/chef/chef-docs/commit/d705baee8d699772692c55fbf565376c6011f16d

**Comment by jamescott** [comment]
Step two: put all the LWRP topics on a single page: https://docs.chef.io/lwrp.html

**Comment by jamescott** [comment]
Step three: add a new HWRP topic: https://docs.chef.io/hwrp.html

**Comment by jamescott** [comment]
Step four: loop in @burtlo and @martinisoft ... it'll be easier to collect feedback and ideas now that all of the info that exists in the lwrp.html or hwrp.html topics.

james

**Comment by martinisoft** [comment]
@jamescott Yes, I've been working with @burtlo to get some quick changes out the door, but we will need a lot more help in the way of majorly updating the Libraries page. We also need some help defining what an HWRP is as that term has been used by Chef, but never defined in the documentation so a newbie could get lost.
